src/main/java/axoloti/utils/Preferences.java

```diff
@@ -75,6 +78,9 @@
     @Element(required = false)
     String themePath;
 
+    @Element(required = false)
+    String Platform;
```

Comment:
unfortunately, as you will see from other fields in the class, this is not the standard in the preferences class, so I followed whats in the class rather than my preference for naming conventions